### DISCUSSION TITLE
[Feature] Auto-play (slideshow) stacked files

### DISCUSSION THREAD

**Discussion by schizovivek** [immich-app/immich#14979] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I've tried to search for items related to stack and didn't find anything so raising as a separate request. 

I've recently discovered the stacking feature and I've setup a few sets as of now. Was wondering if there could be an option to auto-play the files stacked. User could configure the time delay and the UI would then play the slideshow accordingly.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#13404
